proto/buf/alpha/registry/v1alpha1/list.proto

```diff
@@ -0,0 +1,26 @@
+// Copyright 2020-2021 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.registry.v1alpha1;
+
+message ListOptions {
+  // The number of items to return.
+  uint32 page_size = 1;
+  // The first page is returned if this is empty.
+  string page_token = 2;
+  // string value following SQL syntax specifying order for list results.
+  string order_by = 3;
```

Comment:
Following on from Peter's comment, if this is made more explicit, I would argue we could type this field too. This could be an enum, combined with an `bool ascending = 4`. What do you think?

Reply:
Actually we already have reverse, so no need to do the `ascending` part.

Reply:
if we choose to go with an enum do we want it to exhaust every column possibility that we'd want to eventually sort by?

Reply:
I think we can easily support adding things, so we should just add the ones we need right now

Reply:
I agree with @johanbrandhorst that this should be an enum in Protobuf, you can add to enums without breaking compatibility

Reply:
sounds good - enum it is 👍